pipelines/edx/mfe-frontend-app-learning-mitx-staging-ci.yaml

```diff
@@ -0,0 +1,75 @@
+---
+resource_types:
+- name: rclone
+  type: docker-image
+  source:
+    repository: mitodl/concourse-rclone-resource
+    tag: latest
+
+resources:
+- name: mfe-app-learning
+  type: git
+  source:
+    uri: https://github.com/edx/frontend-app-learning.git
+    branch: master
+- name: mfe-app-learning-bucket
+  type: rclone
+  source:
+    config: |
+      [s3-remote]
+      type = s3
+      provider = AWS
+      env_auth = true
+      region = us-east-1
+
+jobs:
+- name: build
+  plan:
+  - get: mfe-app-learning
+    trigger: false
+  - task: npm-install-and-build
+    config:
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          repository: node
+      inputs:
+      - name: mfe-app-learning
+      outputs:
+      - name: mfe-app-learning/dist
+      params:
+        ACCESS_TOKEN_COOKIE_NAME: edx-jwt-cookie-header-payload
+        BASE_URL: https://staging-ci.mitx.mit.edu
+        CSRF_TOKEN_API_PATH: /csrf/api/v1/token
+        LMS_BASE_URL: https://staging-ci.mitx.mit.edu
+        LOGIN_URL: https://staging-ci.mitx.mit.edu/login
+        LOGOUT_URL: https://staging-ci.mitx.mit.edu/logout
+        LOGO_URL: https://raw.githubusercontent.com/mitodl/mitx-theme/master/lms/static/images/logo.png
+        LOGO_TRADEMARK_URL: https://raw.githubusercontent.com/mitodl/mitx-theme/master/lms/static/images/logo.png
+        LOGO_WHITE_URL: https://raw.githubusercontent.com/mitodl/mitx-theme/master/lms/static/images/logo.png
+        FAVICON_URL: https://raw.githubusercontent.com/mitodl/mitx-theme/master/lms/static/images/favicon.ico
+        MARKETING_SITE_BASE_URL: https://staging-ci.mitx.mit.edu
+        ORDER_HISTORY_URL:
+        PUBLIC_PATH: /learn/
+        REFRESH_ACCESS_TOKEN_ENDPOINT: https://staging-ci.mitx.mit.edu/login_refresh
+        SEARCH_CATALOG_URL: https://staging-ci.mitx.mit.edu/courses
+        SITE_NAME: MIT xPRO
+        STUDIO_BASE_URL: https://studio-staging-ci.mitx.mit.edu
+        SUPPORT_URL: https://odl.zendesk.com/hc/en-us/requests/new
+        USER_INFO_COOKIE_NAME: edx-user-info
+        SESSION_COOKIE_DOMAIN: .mitx.mit.edu
```

Comment:
Let's try using the LMS domain so we don't have to worry about sessions bleeding between environments.